mixins/js/ez-tabs.js

```diff
@@ -107,23 +107,25 @@ window.eZ = window.eZ || {};
              *
              * @param {HTMLElement} panel
              * @param {HTMLElement} tabsLabel
+             * @param event {Event} click event
              */
-            _changeTab(panel, tabsLabel) {
-                this._unselectTab();
+            _changeTab(panel, tabsLabel, event) {
+                this._unselectTab(event);
 
                 tabsLabel.classList.add(TAB_IS_SELECTED);
                 panel.classList.add(TAB_IS_SELECTED);
             }
 
             /**
              * Unselects currently selected tabs
+             *
+             * @param event {Event} click event
              */
-            _unselectTab() {
-                const forEach = Array.prototype.forEach;
+            _unselectTab(event) {
+                const tabsContainer = event.target.closest('.ez-tabs');
+                const selectedTabs = tabsContainer.querySelectorAll('.' + TAB_IS_SELECTED);
 
-                forEach.call(this.querySelectorAll('.' + TAB_IS_SELECTED), function (element) {
-                    element.classList.remove(TAB_IS_SELECTED);
-                });
+                [].forEach.call(selectedTabs, (element) => element.classList.remove(TAB_IS_SELECTED));
```

Comment:
I agree with @andrerom on `[].forEach` vs. `Array.prototype.forEach`, especially given this change is completely unrelated to the issue (and useless) so to keep the patch as short as possible, I would only change the first parameter of `forEach.call`. 

that said thanks to the web components polyfill, you can even use `forEach` method directly on NodeList returned by `querySelectorAll`.
(see https://github.com/ezsystems/hybrid-platform-ui-core-components/pull/30#discussion_r129773517)

For fat arrow vs. regular `function`, we tend to use fat arrow to avoid using `bind` but that's fine this way as well.